src/tools/queryDatasource/queryDatasource.test.ts

```diff
@@ -55,19 +58,23 @@ vi.mock('../../restApiInstance.js', () => ({
   ),
 }));
 
-describe('queryDatasourceTool', () => {
```

Comment:
I refactored these tests to mock `getConfig` instead of relying on setting and resetting environment variables.